ledger/streaming/api/src/error.rs

```diff
@@ -36,7 +36,7 @@ pub enum Error {
     BlockValidation(String),
 
     /// SCP Consensus Behind: last externalized block {0} - highest block {1}
-    ConsensusBlocked(SlotIndex, SlotIndex),
+    ConsensusBlocked(BlockIndex, BlockIndex),
```

Comment:
Is there an issue to come back and disambiguate _Slot_ versus _Block_?
Looking at the usage of `SlotIndex` and `consensus/scp/src/slot.rx` there is:
```rust
//! A unit of time during which the nodes agree on transactions.
//!
//! The transactions validated in this slot determine the values to include in
//! the next block appended to the ledger.
```
Which seems to imply that Slot is a unit of the consensus protocol, looking at `scp_validators.rs` where the `ConsensusBlocked` error is created there is:
```rust
                    return future::ready(Some(Ready::Ready(Err(Error::ConsensusBlocked(
                        scp_state.highest_externalized_slot(),
                        scp_state.highest_block_received(),
```
The `highest_externalized_slot()` is raising some concerns as to what the type should really be.
is this supposed to be `ConsensusBlocked(SlotIndex, BlockIndex)` or is `SlotIndex` really an alias for `BlockIndex`?

Reply:
... Except `mc_consensus_scp_types::SlotIndex` exists, so I'll switch back to that and add a TODO